prow/ANNOUNCEMENTS.md

```diff
@@ -5,6 +5,12 @@
 New features added to each component:
  - *June 23rd, 2020* An [hmac](/prow/cmd/hmac) tool was added to automatically reconcile webhooks and hmac
     tokens for the orgs and repos integrated with your prow instance.
+- *June 12, 2020* Added _bucket_ and _folder_ validation features to Deck to restrict artifact requests to only
+     specific storage locations. This feature can be enabled by setting `deck.restrict_storage_paths` in the Prow config.
+     By default, _buckets_ specified in the `plank.decoration_config[*].gcs_configuration.bucket` and 
+     the `pr-logs` and `logs` _folders_ are allowed allowed. _Additional buckets_ can be allowed by adding them to 
```

Comment:
s/allowed allowed/allowed and the folders part isn't accurate anymore

Reply:
Done. (Thanks for noticing; pretty sure this was an oversight after fixing a merge conflict. I've added a Markdown previewer to my IDE now.)